test/unit/domain/consents/revoke.test.ts

```diff
@@ -119,7 +119,7 @@ describe('server/domain/consents/revoke', (): void => {
           .toThrowError(new DatabaseError(partialConsentActiveConflictingInitiatorId.id))
 
         expect(mockConsentUpdate).toHaveBeenCalled()
-        expect(mockLoggerPush).toHaveBeenCalledWith(testError)
```

Comment:
that is strange - we are calling`push` in revoke.ts L82

Reply:
I think you reviewed an old version. It says "Outdated" on the diff. 

Either way, it has been converted to ` expect(mocked(logger.push)).toHaveBeenCalledWith({ consent: expect.objectContaining(partialConsentActiveConflictingInitiatorId) })` so we are indeed expecting a push. Just using the newer syntax that you introduced in your logger PR.